internal/services/databricks/resourceids.go

```diff
@@ -0,0 +1,4 @@
+package databricks
```

Comment:
this file shouldn't be needed anymore, since it's available within `go-azure-sdk`?